src/setup.erl

```diff
@@ -134,6 +140,10 @@ enable_cluster_int(Options, no) ->
             config:set("httpd", "bind_address", binary_to_list(NewBindAddress))
     end,
 
+    NodeCount = couch_util:get_value(node_count, Options),
+    ok = require_clustersize(NodeCount),
+    config:set_integer("cluster", "n", NodeCount),
```

Comment:
this is the issue we are trying to fix: https://issues.apache.org/jira/browse/COUCHDB-2594